lib/Agrammon/DB/Dataset.pm6

```diff
@@ -65,7 +65,7 @@ class X::Agrammon::DB::Dataset::InstanceRenameFailed is Exception {
     }
 }
 
-#| Error when instance couldn't be renamed.
+#| Error when input comment couldn't be renamed.
```

Comment:
Is "renamed" really accurate here? It's "save" in the message and "store" in the class name.

Reply:
Fixed